benchmark_runner/common/template_operations/templates/vdbench/vdbench_data_template.yaml

```diff
@@ -22,10 +22,10 @@ template_data:
       DIRECTORIES: 600
       FILES_PER_DIRECTORY: 10
       SIZE_PER_FILE: 10
-      limits_cpu: 20m
+      limits_cpu: 2
       limits_memory: 2Gi
-      requests_cpu: 2
-      requests_memory: 2Gi 
+      requests_cpu: 500m
+      requests_memory: 512Mi
```

Comment:
Mem request seems pretty low, we tend to use 32GB+ in other cases.  Also would recommend mem req = mem limit like other testcases, otherwise pods can burst to use more than vms.

Reply:
I have updates it accordingly for perf ci :
```
limits_cpu: 2
limits_memory: 2Gi
requests_cpu: 2
requests_memory: 2Gi
sockets: 2
```